zlib 1.2.6 changed the semantics of gzeof function which breaks xetex
and probably other programs (good move for a minor version upgrade!)
This patch was made by Dan Ports to fix xetex
---
 texk/web2c/xetexdir/xetex.ch |    1 -
 1 file changed, 1 deletion(-)

Index: texlive-bin-2009/texk/web2c/xetexdir/xetex.ch
===================================================================
--- texlive-bin-2009.orig/texk/web2c/xetexdir/xetex.ch	2012-02-14 08:12:16.892244044 +0900
+++ texlive-bin-2009/texk/web2c/xetexdir/xetex.ch	2012-02-14 08:12:17.600244497 +0900
@@ -6951,7 +6951,6 @@
 @y
 undump_int(x);
 if (x<>69069) then goto bad_fmt;
-if not w_eof(fmt_file) then goto bad_fmt
 @z
 
 @x
